inc/class.client.php

```diff
@@ -63,6 +66,16 @@ public function __construct() {
 		return true;
 	}
 
+	function enqueue_displayformat_scripts() {
+
+		wp_register_script('taxopress-frontend.js', STAGS_URL . '/assets/frontend/js/frontend.js', array('jquery'), STAGS_VERSION);
+		wp_register_style('taxopress-frontend.css', STAGS_URL . '/assets/frontend/css/frontend.css', array(), STAGS_VERSION, 'all');
```

Comment:
Same as above, the register name should be taxopress-frontend-css and not taxopress-frontend.css

Reply:
Okay, done that.
Thank you